lib/api/wpc_proto.h

```diff
@@ -13,9 +13,33 @@
 /*
  * Maximum size of a response. It can be used as an hint
  * to give a big enough buffer to @ref WPC_Proto_handle_request
- * or @WPC_Proto_get_current_event_status
+ * note : it should be higher than this response size list, as it's a wp_GenericMessage
+ * wp_GetConfigsResp_size
+ * wp_GetGwInfoResp_size
+ * wp_SetConfigResp_size
+ * wp_SendPacketResp_size
+ * wp_GetScratchpadStatusResp_size
+ * wp_ProcessScratchpadResp_size
+ * wp_SetScratchpadTargetAndActionResp_size
+ * wp_UploadScratchpadResp_size
  */
-#define WPC_PROTO_MAX_RESPONSE_SIZE 512
+#define WPC_PROTO_MAX_RESPONSE_SIZE 450 
```

Comment:
Should we add a static assert that all the listed response size feat to this value?

Ideally a MAX(val1, val2, val3...) would be good but such macro doesn't exist